src/services/loginUser.ts

```diff
@@ -0,0 +1,21 @@
+import bcrypt from 'bcryptjs';
+import { Request, Response } from 'express';
+
+import User from '../models/user.js';
+import generateAccessToken from '../services/generateAccessToken.js';
+
+const loginUser = async (req: Request, res: Response) => {
```

Comment:
Your service methods should only accept the necessary payload. In this case, the `loginUser` method should accept an object with just `email` and `password` and return just a `token`. Otherwise, throw a custom [exception](https://bobbyhadz.com/blog/typescript-extend-error-class)

```
const loginUser = (payload:{email:string,password:string}) =>{
...
if(!user){
 throw new AppException("User not found");
}
...
return 'token'
}
```